tests/e2e_tests/inference_endpoints/test_inference_endpoints.py

```diff
@@ -282,6 +283,7 @@ def test_prediction_with_certificates_v2():
 
 
 def test_version_not_served():
+    time.sleep(30)
```

Comment:
It's really necessary here to put this sleep? It's seems that tests checks if model version is not available, so we do not have to require fully readiness for this endpoint.